### ISSUE TITLE
option to set the next button text

### ISSUE THREAD

**Issue by ppazos** [easelinc/tourist#39]
Add support to set the next button text of each step.

This is a pretty basic option that I can't find on the documentation. Also this is needed to support I18N provided by the system.

**Comment by mcolyer** [comment]
Hi @ppazos we actually aren't investing in this project these days and unfortunately, the project doesn't currently offer the ability to change that text.

If you're interested in implementing those changes I could review them.

**Comment by ppazos** [comment]
Thanks @mcolyer  yes I would like to contribute. 
I'm not sure how it works internally, I was thinking of a workaround, getting the button using the class and changing the inner text via JS. The only issue with this approach is that the classes for the Next button and the Finish button are the same and it seems I can't get it the current tooltip is the last one or not.

A better way would be to specify the button text as an option on each step, and that can be a variable I can set form the current locale, so the text is customizable and I18N. But I'm not sure where to start on this approach.

Any pointers are welcome.

**Comment by mcolyer** [comment]
I believe you'll want to provide configurability to [this array](https://github.com/easelinc/tourist/blob/a31cf497cc7af8173513ea3f57f760894d5262cb/src/tip/base.coffee#L21).

**Comment by ppazos** [comment]
@mcolyer yes! I have an interface in EN and ES, and for Spanish I see "Next step", I need that to be "Próximo paso".

The I18N is done on the backend so I don't need both EN and ES on the UI at the same time, jjust set all texts for ES or set all for EN (or leave by default).

My hack:

```
      $('#tour-btn').on('click', function(){
        TOUR.start();
        $('.tour-buttons > button')[0].innerText = '${message(code:"app.get_started.tour.next")}';
      });
```
In my app, the message function is what resolves against the current locale.

**Comment by ppazos** [comment]
@mcolyer I believe what I'm seeing is really 

`a.prototype.nextButtonTemplate = '<button class="btn btn-primary btn-sm pull-right tour-next">Next step →</button>',`

not "Next this:"

` actionLabels: ['Do this:', 'Then this:', 'Next this:']`


Also my solution above only changed the first button :(


Solution:

```
Tourist.Tip.Base.prototype.nextButtonTemplate = Tourist.Tip.Base.prototype.nextButtonTemplate.replace('Next step →', '${message(code:"app.get_started.tour.next").decodeHTML()}');
Tourist.Tip.Base.prototype.finalButtonTemplate = Tourist.Tip.Base.prototype.finalButtonTemplate.replace('Finish up', '${message(code:"app.get_started.tour.finish").decodeHTML()}');

TOUR = new Tourist.Tour({......
```

**Comment by mcolyer** [comment]
> Solution:

I wouldn't be able to merge that solution, as it's not really the right way to integrate it. If you're able to create an API in Tourist to modify `actionLabels`, I'd consider merging it. 

If you don't want to do that, that's okay too.
